other/appfilter.xml

```diff
@@ -2923,7 +2927,7 @@
   <item component="ComponentInfo{io.github.robertaguilera712.cisojr4droid/io.github.robertaguilera712.cisojr4droid.activities.MainActivity}" drawable="cisojr4droid"/>
 
   <!-- Citi Mobile -->
-  <item component="ComponentInfo{com.citi.citimobile/com.citi.mobile.pt3.GlobalPhoneActivity}" drawable="citi_mobile"/>
+  <item component="ComponentInfo{com.citi.ci	bile/com.citi.mobile.pt3.GlobalPhoneActivity}" drawable="citi_mobile"/>
```

Comment:
Mistake? This will break the icon for "Citi Mobile"

Reply:
More typos... I'll fix them

Thanks for looking out